app/main.py

```diff
@@ -1,3 +1,49 @@
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+
+    def __init__(self, x, y):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        return Vector(round(self.x + other.x, 2), round(self.y + other.y, 2))
+
+    def __sub__(self, other):
+        return Vector(round(self.x - other.x, 2), round(self.y - other.y, 2))
+
+    def __mul__(self, other):
+        if type(other) == Vector:
+            return (self.x * other.x) + (self.y * other.y)
+        else:
```

Comment:
remove unnecessary `else`

Reply:
Привет. 
Вчера я удалил Python 3.8 и установил 3.10. Из за этого возникли проблемы с коммитом.

hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Я понял свою ошибку по поводу лишнего else. 
Можешь ли ты принять эту задачу в таком виде?
Или как мне быть ?
Я уже удалил предыдущую папку с заданием и клонировал ее заново, но не помогло.

Reply:
okay, I've approved, maybe you should delete your repository on GitHub, and do fork again